tests/test_applications.py

```diff
@@ -98,7 +103,7 @@ def custom_ws_exception_handler(websocket: WebSocket, exc: CustomWSException):
     ]
 )
 
-exception_handlers = {
+exception_handlers: Any = {
```

Comment:
Let's try to minimize the `Any`s. Can we remove this one?

Reply:
If i remove `any` from `exception_handlers: Any` i get the following error:

`error: Argument "exception_handlers" to "Starlette" has incompatible type "Dict[object, function]"; expected "Optional[Mapping[Any, Union[Callable[[Request, Exception], Union[Response, Awaitable[Response]]], Callable[[WebSocket, Exception], Awaitable[None]]]]]"`

If I add `exception_handlers: ExceptionHandlers` i get the following errors: 

`Dict entry 0 has incompatible type "int": "Callable[[Request, HTTPException], Coroutine[Any, Any, JSONResponse]]"; expected "Any": "Union[Callable[[Request, Exception], Union[Response, Awaitable[Response]]], Callable[[WebSocket, Exception], Awaitable[None]]]"`

`Dict entry 1 has incompatible type "int": "Callable[[Request, HTTPException], Coroutine[Any, Any, JSONResponse]]"; expected "Any": "Union[Callable[[Request, Exception], Union[Response, Awaitable[Response]]], Callable[[WebSocket, Exception], Awaitable[None]]]"`

`Dict entry 2 has incompatible type "Type[HTTPException]": "Callable[[Request, HTTPException], Coroutine[Any, Any, JSONResponse]]"; expected "Any": "Union[Callable[[Request, Exception], Union[Response, Awaitable[Response]]], Callable[[WebSocket, Exception], Awaitable[None]]]"`

`Dict entry 3 has incompatible type "Type[CustomWSException]": "Callable[[WebSocket, CustomWSException], None]"; expected "Any": "Union[Callable[[Request, Exception], Union[Response, Awaitable[Response]]], Callable[[WebSocket, Exception], Awaitable[None]]]"`

If I add `exception_handlers: Mapping[Any, ExceptionHandler]` as I saw in `starlette/applications.py` i get the same errors as above.

If I add `exception_handlers: dict[
            typing.Any, typing.Callable[[Request, Exception], Response]
        ]` I get the following errors:

`Dict entry 0 has incompatible type "int": "Callable[[Request, HTTPException], Coroutine[Any, Any, JSONResponse]]"; expected "Any": "Callable[[Request, Exception], Response]"`

`Dict entry 1 has incompatible type "int": "Callable[[Request, HTTPException], Coroutine[Any, Any, JSONResponse]]"; expected "Any": "Callable[[Request, Exception], Response]"`

`Dict entry 2 has incompatible type "Type[HTTPException]": "Callable[[Request, HTTPException], Coroutine[Any, Any, JSONResponse]]"; expected "Any": "Callable[[Request, Exception], Response]"`

`Dict entry 3 has incompatible type "Type[CustomWSException]": "Callable[[WebSocket, CustomWSException], None]"; expected "Any": "Callable[[Request, Exception], Response]"`

I have searched everywhere related to `exception_handlers` type hints but this is all I could find and it looks like nothing is working. That's why I have added `any` there. Do you have any idea on how to fix this?

Reply:
I'd prefer the `type: ignore` than `Any`.